lte/gateway/docker/services/python/Dockerfile

```diff
@@ -117,12 +119,12 @@ RUN echo "deb https://packages.fluentbit.io/ubuntu/focal focal main" > /etc/apt/
 RUN wget -qO - https://packages.fluentbit.io/fluentbit.key | apt-key add -
 
 RUN apt-get update && apt-get install -y \
-  td-agent-bit \
+  bcc-tools \
   libopenvswitch \
   openvswitch-datapath-dkms \
   openvswitch-common \
   openvswitch-switch \
-  bcc-tools \
+  td-agent-bit \
   wireguard
```

Comment:
Do we want a && rm -rf /var/lib/apt/lists/* here?

Reply:
I added it.